upload/library/Sidane/Threadmarks/Model/Post.php

```diff
@@ -12,6 +12,7 @@ public function preparePost(array $post, array $thread, array $forum, array $nod
     $post['canAddThreadmarks'] = $threadmarkmodel->canAddThreadmark($post, $thread, $forum, $null, $nodePermissions, $viewingUser);
     $post['canEditThreadmarks'] = $threadmarkmodel->canEditThreadmark($post, $thread, $forum, $null, $nodePermissions, $viewingUser);
     $post['canDeleteThreadmarks'] = $threadmarkmodel->canDeleteThreadmark($post, $thread, $forum, $null, $nodePermissions, $viewingUser);
+    $post['canViewThreadmarks'] = $threadmarkmodel->canViewThreadmark($post, $thread, $forum, $null, $nodePermissions, $viewingUser);
```

Comment:
The arguments passed into this method are wrong. It's raising a `Undefined index: node_id` error.

It should be `$threadmarkmodel->canViewThreadmark($thread, $null, $nodePermissions, $viewingUser);`